app/models/email_template.rb

```diff
@@ -0,0 +1,17 @@
+class EmailTemplate < ApplicationRecord
+  before_save :ensure_exactly_one_active_per_kind
+
+  enum kind: { Anmeldung: 0, Probezeit: 1, Begleitung: 2 }
```

Comment:
I suggest using enum keys that match the values on the other side. Eg. `Probezeit` -> `probation_period`

The Translation issue, with Enum fields, I've just discovered that it is actually many times easyer as we knew so far. 

If in the form:

```ruby

f.input :kind, collection: EmailTemplate.kinds.keys.map(&:to_sym), ....
```
In the translation yaml:

```yaml
simple_form:
  options:
    email_template:
      kind:
        probation_period: Probezeit 
```
This way it translates perfectly well. It is the good (rather bad) old fuzziness simple form has with the collection keys. It will only translate them if passed as Keys, and it doesn't care if its enum or whatever else. `collection: [:a, :b] ` passed to input will allways look up the options in the simple_form.options.model_name.field.key

Reply:
If feasible you could also cherry-pick [this](https://github.com/panter/aoz-003/pull/359/files#diff-1717d3e3835cc67fc19f0620ff5bc59a) here if you want.
I that case you could do exactly like [i did here](https://github.com/panter/aoz-003/blob/f11f43679daa87f37f9fcb8d904cee94ab637f56/app/views/volunteers/_form.html.slim#L7).